CATALOG.md

```diff
@@ -0,0 +1,42 @@
+# Build Commands Library
```

Comment:
Let's keep consistent naming between filename and title. "Catalog" everywhere or "Library" everywhere or "Example" everywhere. All are fine by me, your choice